docs/topics/db/models.txt

```diff
@@ -1238,6 +1238,11 @@ number of proxy models that share a common non-abstract parent class.
     In earlier versions, a proxy model couldn't inherit more than one proxy
     model that shared the same parent class.
 
+.. note::
+
+    The ``ordering`` and ``get_latest_by`` meta attributes are still inherited
```

Comment:
My suggestion was to explain why the behavior applies to these Meta attributes but not others. I guess you might need to do some research about that.